components/atom/switch/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@schibstedspain/sui-atom-switch",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -Rf ./lib && mkdir -p ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "1",
+    "@s-ui/react-atom-label": "^1.1.0"
```

Comment:
Just use the major, as we're folling Compatible Version:

`"@s-ui/react-atom-label": "1"`